docs/UserGuide.md

```diff
@@ -35,100 +39,194 @@
+> * The index refers to the index number shown in the current listing.<br>
+> * The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `update todo 3 description Avengers`<br>
+   Selects the 3rd todo task from the displayed list and changes the task description to �Avengers�
+* `update todo 4 date 11 Nov`<br>
+   Selects the 4th todo task from the displayed list and changes the type of task from todo to scheduled. 
+* `update schedule 5 time 8:30pm`<br>
+   Selects the 5th schedule task from the displayed list and changes the time to 8:30pm. <br>
+<img src="images/UI_Prototype/WhatNowUI-UpdateSchedule-Colour.png" width="600"> 
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+#### Undoing the previous action : `undo`
+There can be multiple undos to revert to the previous state.<br>
+Format: `undo`  
 
 Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+* `undo`<br>
+   Undo the previous action <br>
+<img src="images/UI_Prototype/WhatNowUI-Undo-Colour.png" width="600">
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+#### Redoing the previous action : `redo`
+There can be multiple redos to revert to the previous state.<br>
+Format: `redo`  
 
 Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
-
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
+* `redo`<br>
+   Redo the previous action <br>
+<img src="images/UI_Prototype/WhatNowUI-Redo-Colour.png" width="600">
+
+
+#### Finding tasks containing any keyword in their description: `find`
+Search for all tasks whose description contains any of the given keywords.<br>
+Format: `find KEYWORD [MORE_KEYWORDS]`
+
+> * The search is case sensitive. e.g `Read` will not match `read`
+> * The order of the keywords does not matter. e.g. `Read books` will match `books Read`
+> * Only the task description is searched.
```

Comment:
We did not want to find by date/time/priority because if you search for 10 Oct 2016 then it will show you all the tasks on 10 Oct 2016 right? Then isn't that the same as just list date 10 Oct 2016 which will also show all the tasks on 10 Oct 2016?

Same for time and priority. And we think that using the command list to do that makes more sense then using search. Unless you mean you can find 6 by date then it will show all the dates with a 6 in it. Then find will make more sense. And we will think about this again. Thanks.